ERCS/erc-XXXX.md

```diff
@@ -0,0 +1,254 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 7754
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.